### PR TITLE
Add FBT for logs

### LINKED ISSUE #503: Finish remaining FBT sheets
#### Issue details
We should complete the remaining FBT sheets.

Missing:
- Replication Details
- Logs

Any comments on the process? Should we adapt or change something?

@floreks , @digitalfishpond @maciaszczykm @f-higashi 
please pick

### PR DESCRIPTION
**Pull request by maciaszczykm**
Fixes #503 issue.

---
This change is [Review on Reviewable](https://reviewable.io/reviews/kubernetes/dashboard/513)

### CODE REVIEW

**Comment by maciaszczykm** [comment]
@cheld Could you take a look?

File: docs/test/replication-controller-list.md [hunk 1]
```diff
@@ -9,25 +9,19 @@ Cluster with few deployed applications, where:
...
-|--------------------------------------|-------------------------------|-------------------------------------|----------------|--------------|-----------------|---------------------------|------------|--------|---------------------------------------------------------------------------------------------|
-| Replication controller count on page | 1                             | odd                                 | even           | 20-30        |                 |                           |            |        |                                                                                             |
-| Screen size                          | normal                        | mobile                              | narrow         | wide         | active resizing |                           |            |        |                                                                                             |
```
Comment (cheld) [inline docs/test/replication-controller-list.md]: For RC list page, the layouting is a very important feature. That's why we should test screen sizes in very detail. I don't see it for logs page. I would leave it to the general factor 'screen size'

File: docs/test/replication-controller-list.md [hunk 2]
```diff
@@ -9,25 +9,19 @@ Cluster with few deployed applications, where:
...
-| Delete replication controller action | normal                        | with checkbox checked               | cancel         |              |                 |                           |            |        |                                                                                             |
-| Concurrency                          | delete replication controller | edit pod count                      |                |              |                 |                           |            |        |                                                                                             |
+| Factor                        | 1               | 2                   | 3            | 4           | 5    | 6               | 7 | 8 | Comment |
+|-------------------------------|-----------------|---------------------|--------------|-------------|------|-----------------|---|---|---------|
+| Screen size                   | normal          | mobile              | narrow       | middle size | wide | active resizing |   |   |         |
```
Comment (cheld) [inline docs/test/replication-controller-list.md]: For RC list page, the layouting is a very important feature. That's why we should test screen sizes in very detail. I don't see it for logs page. I would leave it to the general factor 'screen size'

File: docs/test/replication-controller-list.md [hunk 3]
```diff
@@ -9,25 +9,19 @@ Cluster with few deployed applications, where:
...
+| Pods count                    | 1               | odd                 | even         | 20-30       |      |                 |   |   |         |
```
Comment (cheld) [inline docs/test/replication-controller-list.md]: Again odd and even are important factors to test the layouting on RC list page. But I do not consider these as important factors for log page

File: docs/test/replication-controller-list.md [hunk 4]
```diff
@@ -9,25 +9,19 @@ Cluster with few deployed applications, where:
...
+| Container count               | 0               | 0-10                | more than 10 |             |      |                 |   |   |         |
```
Comment (cheld) [inline docs/test/replication-controller-list.md]: I assume 0 not possible :). I would strip down to two factors 1 and more than 10

File: docs/test/replication-controller-list.md [hunk 5]
```diff
@@ -9,25 +9,19 @@ Cluster with few deployed applications, where:
...
+| Replication Controller status | valid state     | invalid error state |              |             |      |                 |   |   |         |
```
Comment (cheld) [inline docs/test/replication-controller-list.md]: Does this have an effect on logs page?

**Comment by cheld** [comment]
Think about the following factors:
- log size small, large (1 GB). I am not sure what happens. Does the browser blow up? Does the API server cut somehow?
- max. name length for pod and container (we had bugs with layout!)
- concurrency. e.g. delete the pod that is currently displayed (because of hpa scale-down)
- log lines, e.g. very long lines 

That's all I could think off. Please investigate and add more factors.

File: docs/test/replication-controller-list.md (hunk 2 above)
Comment (maciaszczykm) [inline docs/test/replication-controller-list.md]: I can move it to general factor, but I thought, that general factors should be the same for all sheets. Regarding screen size factor I think it's important due to toolbar and it's behaviour on smaller screens. I agree, that there are many cases for this factor, but this test should be quick anyways.

File: docs/test/replication-controller-list.md (hunk 3 above)
Comment (maciaszczykm) [inline docs/test/replication-controller-list.md]: Agree, done.

File: docs/test/replication-controller-list.md (hunk 4 above)
Comment (maciaszczykm) [inline docs/test/replication-controller-list.md]: You're right. Done.

File: docs/test/replication-controller-list.md (hunk 5 above)
Comment (maciaszczykm) [inline docs/test/replication-controller-list.md]: Yes, it could have impact, and because of it I've decided to put it there.

**Comment by maciaszczykm** [comment]
>Think about the following factors:

>- log size small, large (1 GB). I am not sure what happens. Does the browser blow up? Does the API server cut somehow?
- max. name length for pod and container (we had bugs with layout!)
- concurrency. e.g. delete the pod that is currently displayed (because of hpa scale-down)
- log lines, e.g. very long lines

>That's all I could think off. Please investigate and add more factors.

I've added logs size, pod and container name factors. Concurrency won't have any impact, because we do not refresh view asynchronously. Regarding log lines length, we do not have any impact on it, and I think when we will be testing logs with large size (1GB), then we'll see all kinds of logs line length.

**Comment by cheld** [comment]
About concurrency: 

Steps:
1. Open Logs => second browser tab opens
2. Switch browser tab
3. delete pod
4. switch to 2nd tab that shows the log
5. press refresh

=> 

Internal Server Error (500)

pods "frontend-iei5o" not found


Do we consider this as a bug? Maybe ok for the moment. I am ok with skipping concurrency.

**Comment by cheld** [comment]
LGTM, I merge as I do not expect further comments.

**Pull request merged**